fastlane_core/lib/fastlane_core/print_table.rb

```diff
@@ -7,7 +7,14 @@ class << self
       def print_values(config: nil, title: nil, hide_keys: [], mask_keys: [])
         require 'terminal-table'
 
-        options = config.nil? ? {} : config.values(ask: false)
+        options = {}
+        unless config.nil?
+          if config.kind_of?(FastlaneCore::Configuration)
+            options = config.values(ask: false)
+          else
+            options = config.values if options.count == 0
```

Comment:
I think `options.count` is always equal to zero here.

If `config` is a hash, like in the test, `config.values` will return an array. However, I think `collect_rows` expects a hash. Is this intended?